fix: update dependencies for Astro v6

Installing Astro v6 works, but only if no MDX support is installed. Somehow the dependencies fail until I remove MDX support. Then on reinstalling a couple of version issues crop up:

```plaintext
➜ npm install @astrojs/mdx
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: astro-meta-tags@0.4.0
npm error Found: astro@6.0.4
npm error node_modules/astro
npm error   peer astro@">=3.x || >=4.0.0-beta || >=5.x" from @sentry/astro@10.43.0
npm error   node_modules/@sentry/astro
npm error     peer @sentry/astro@">=7.84.0" from @spotlightjs/astro@3.2.6
npm error     node_modules/@spotlightjs/astro
npm error       @spotlightjs/astro@"3.2.6" from the root project
npm error     @sentry/astro@"10.43.0" from the root project
npm error   peer astro@">=4.7.0" from @spotlightjs/astro@3.2.6
npm error   node_modules/@spotlightjs/astro
npm error     @spotlightjs/astro@"3.2.6" from the root project
npm error   3 more (astro-expressive-code, the root project, @astrojs/mdx)
npm error
npm error Could not resolve dependency:
npm error peer astro@"^4.0.0 || ^5.0.0-beta.0" from astro-meta-tags@0.4.0
npm error node_modules/astro-meta-tags
npm error   astro-meta-tags@"0.4.0" from the root project
npm error
npm error Conflicting peer dependency: astro@5.18.1
npm error node_modules/astro
npm error   peer astro@"^4.0.0 || ^5.0.0-beta.0" from astro-meta-tags@0.4.0
npm error   node_modules/astro-meta-tags
npm error     astro-meta-tags@"0.4.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry this command with --force or --legacy-peer-deps to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /home/patrick/.npm/_logs/2026-03-14T22_24_54_425Z-eresolve-report.txt
npm error A complete log of this run can be found in: /home/patrick/.npm/_logs/2026-03-14T22_24_54_425Z-debug-0.log
```

Removing astro-meta-tags solves the issue, but, ehm, removes astro-meta-tags.

--- a/.github/workflows/format-check.yml
+++ b/.github/workflows/format-check.yml
@@ -21,10 +21,10 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v4
         with:
-          version: 8
+          version: 9
 
       - name: Install dependencies
         run: pnpm install
 
       - name: Check formatting
-        run: npx prettier . --check
+        run: pnpm exec prettier . --check

--- a/packages/astro-meta-tags/package.json
+++ b/packages/astro-meta-tags/package.json
@@ -36,6 +36,6 @@
     "typescript": "^5.3.3"
   },
   "peerDependencies": {
-    "astro": "^4.0.0 || ^5.0.0-beta.0"
+    "astro": ">=4.0.0"
   }
 }

--- a/packages/astro-meta-tags/package.json
+++ b/packages/astro-meta-tags/package.json
@@ -1,6 +1,6 @@
 {
   "name": "astro-meta-tags",
-  "version": "0.4.0",
+  "version": "0.4.1",
   "author": "Patrick Arminio",
   "repository": "github:patrick91/astro-meta-tags",
   "license": "MIT",

--- a/packages/demo/src/pages/index.astro
+++ b/packages/demo/src/pages/index.astro
@@ -21,14 +21,14 @@ import { SITE_TITLE, SITE_DESCRIPTION } from "../consts";
       <p>
         Welcome to the official <a href="https://astro.build/">Astro</a> blog starter
         template. This template serves as a lightweight, minimally-styled starting
-        point for anyone looking to build a personal website, blog, or portfolio
-        with Astro.
+        point for anyone looking to build a personal website, blog, or portfolio with
+        Astro.
       </p>
       <p>
         This template comes with a few integrations already configured in your
         <code>astro.config.mjs</code> file. You can customize your setup with
-        <a href="https://astro.build/integrations">Astro Integrations</a> to add
-        tools like Tailwind, React, or Vue to your project.
+        <a href="https://astro.build/integrations">Astro Integrations</a> to add tools
+        like Tailwind, React, or Vue to your project.
       </p>
       <p>Here are a few ideas on how to get started with the template:</p>
       <ul>
@@ -53,8 +53,7 @@ import { SITE_TITLE, SITE_DESCRIPTION } from "../consts";
         Have fun! If you get stuck, remember to <a
           href="https://docs.astro.build/"
           >read the docs
-        </a> or <a href="https://astro.build/chat">join us on Discord</a> to ask
-        questions.
+        </a> or <a href="https://astro.build/chat">join us on Discord</a> to ask questions.
       </p>
       <p>
         Looking for a blog template with a bit more personality? Check out <a

